plugins/scaffolder-backend-module-gitlab/src/actions/createGitlabProjectAccessTokenAction.examples.test.ts

```diff
@@ -73,231 +75,127 @@ describe('gitlab:projectAccessToken:create examples', () => {
   });
 
   it('Create a GitLab project access token with minimal options.', async () => {
-    const fetchMock = jest.spyOn(global, 'fetch');
-    const mockResponse = {
-      token: 'mock-access-token',
-    };
-
-    fetchMock.mockResolvedValue({
-      json: async () => mockResponse,
-    } as Response);
-
-    jest.mock('../util', () => ({
-      getToken: jest.fn().mockReturnValue({
-        token: 'mock-api-token',
-        integrationConfig: { config: { baseUrl: 'https://api.gitlab.com' } },
-      }),
-    }));
+    mockGitlabClient.ProjectAccessTokens.create.mockResolvedValue({
+      token: 'TOKEN',
+      username: 'User',
+    });
 
     const input = yaml.parse(examples[0].example).steps[0].input;
-
     await action.handler({
       ...mockContext,
       input,
     });
 
-    expect(fetchMock).toHaveBeenCalledWith(
-      'https://gitlab.com/api/v4/projects/456/access_tokens',
+    expect(mockGitlabClient.ProjectAccessTokens.create).toHaveBeenCalledWith(
+      '456',
+      'tokenname',
+      ['read_repository'],
       {
-        method: 'POST',
-        headers: {
-          'PRIVATE-TOKEN': 'tokenlols',
```

Comment:
Wondering a bit about this custom header, which used to be tested, but now is not tested anymore. Is there a way to add custom headers to the new `@gitbeaker/rest`?